server/tests/flight.test.js

```diff
@@ -0,0 +1,28 @@
+import request from 'supertest-as-promised';
```

Comment:
we should make sure we keep static data files for the db data mocking in a well organized folder structure.